lib.go

```diff
@@ -0,0 +1,20 @@
+// Package sharedlib a package for developers who prefer github
+package sharedlib
+
+const currentVersion = "1.0.0"
+
+// Version returns the library name, please change it for every release. Just
+// for to show changes over time
+func Version() string {
+	return currentVersion
+}
+
+// Name returns the library name
+func Name() string {
+	return "sharedlib"
+}
+
+// Hi let's be friends
+func Hi(name string) string {
+	return "👋  Hi " + name
+}
```

Comment:
thoughts: i _feel_ like we should try to end with a newline in every file

Reply:
does `go fmt` enforce that? If so, we should add it as an action!

Reply:
I can/will add it with .editorconfig, I always customize my projects with it to complement `goimports`